Cargo.toml

```diff
@@ -3,21 +3,21 @@ name = "template-benchmarks-rs"
 version = "0.1.0"
 authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
 build = "src/build.rs"
-edition = "2018"
+edition = "2021"
 
 [dependencies]
 askama = { git = "https://github.com/djc/askama", branch = "main" }
-criterion = "0.4"
+criterion = { version = "*", features = ["html_reports"] }
```

Comment:
I don't want `*` dependencies for these dependencies.

I'd also like a separate commit just for updating criterion.

Reply:
Done, only added criterion `html_reports` feature to generate report automatically.